docs/index.md

```diff
@@ -779,73 +779,77 @@ Mocha supports the `err.expected` and `err.actual` properties of any thrown `Ass
 
 ## Usage
 
-```console
+<!-- usage -->
+
+```plain
```

Comment:
`console` is more appropriate, no?

Reply:
No:

![image](https://user-images.githubusercontent.com/924465/49253357-52f55f80-f3db-11e8-8734-4688da3ebe18.png)

Reply:
why? you're displaying console output.

Reply:
because of the wacko highlighting (my console doesn't highlight like that).  using `plain`:

![image](https://user-images.githubusercontent.com/924465/49256869-4bd34f00-f3e5-11e8-9f5a-429bb438a823.png)